.github/workflows/migrate.yaml

```diff
@@ -0,0 +1,58 @@
+name: Trigger Database Migrations
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Why not just do merges to main?

Reply:
I do not know how to get the name of the branch that is being merged if it triggers on pushes to `main`. As far as I can tell, only the commit SHA is available in that case. From the PR trigger, this is easily accessible information. (If you have a suggestion on how to do this, I can incorporate it.)